proxy_network.c

```diff
@@ -1209,6 +1209,8 @@ static int _reset_bad_backend(mcp_backend_t *be, enum proxy_be_failures err) {
     STAILQ_INIT(&be->io_head);
     be->io_next = NULL; // also reset the write offset.
 
+    LOGGER_LOG(NULL, LOG_PROXYEVENTS, LOGGER_PROXY_BE_ERROR, NULL, proxy_be_failure_text[err], be->name, be->port, be->rbuf, be->rbufused);
```

Comment:
Is it possible to also log how many requests were failed by this event?

Reply:
Maybe..

Reply:
In practice, it would help a lot in correlating client and backend issues. Also provide data on the impact of proxy events.

Reply:
now prints queue depth.